spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/security/IapAuthenticationAutoConfigurationTests.java

```diff
@@ -44,15 +58,15 @@
 
 /**
  * @author Elena Felder
- * @since 1.1
```

Comment:
Why remove?

Reply:
I noticed that the other tests don't add it. I'll put them back!

Reply:
We don't use `@since` on those classes which were written in version `1.0` since the project start.